a2ml/server/README.md

```diff
@@ -0,0 +1,39 @@
+# Development
+
+## Deps
+
+Uses these packages:
+```
+asyncio
+asyncio-redis
+fastapi
+gevent
+jsonpickle
+redis
+uvicorn
+```
+
+## Test
+
+Run dev server:
+```
+cd a2ml/server
+uvicorn server:app --reload
```

Comment:
The server should be just another a2ml command. So to run it:

```
a2ml server [options]
```

Reply:
ok